src/themes/components/checkbox.ts

```diff
@@ -0,0 +1,49 @@
+import { SystemStyleObject } from '@chakra-ui/theme-tools';
```

Comment:
[Optional] you could leverage `@chakra-ui/anatomy` to get better typing of ll the parts of the multipart component